mmv1/products/accesscontextmanager/AccessLevel.yaml

```diff
@@ -0,0 +1,284 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Resource
+  # This is the singular of `AccessLevels`, any changes here should be made to `AccessLevels` as well
+  name: 'AccessLevel'
+  # This is an unusual API, so we need to use a few fields to map the methods
+  # to the right URL.
+  # create_url is the Create URL
+  # base_url is the Get and Delete and Patch URL. It is empty on purpose.
+  # List won't work yet. It should share a URL with Create.
+  create_url: "{{parent}}/accessLevels"
+  base_url: ""
+  self_link: "{{name}}"
+  update_verb: :PATCH
+  update_mask: true
+  references: !ruby/object:Api::Resource::ReferenceLinks
+    guides:
+      'Access Policy Quickstart': 'https://cloud.google.com/access-context-manager/docs/quickstart'
+    api: 'https://cloud.google.com/access-context-manager/docs/reference/rest/v1/accessPolicies.accessLevels'
+  description: |
+    An AccessLevel is a label that can be applied to requests to GCP services,
+    along with a list of requirements necessary for the label to be applied.
+  docs: !ruby/object:Provider::Terraform::Docs
+    warning: |
+      If you are using User ADCs (Application Default Credentials) with this resource,
+      you must specify a `billing_project` and set `user_project_override` to true
+      in the provider configuration. Otherwise the ACM API will return a 403 error.
+      Your account must have the `serviceusage.services.use` permission on the
+      `billing_project` you defined.
+  autogen_async: true
+  id_format: "{{name}}"
+  import_format: ["{{name}}"]
+  examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "access_context_manager_access_level_basic"
```

Comment:
Should we move the examples into the same folder as part of this migration, or is that inflating the scope do you think? No need to do so in this specific PR if we decide to, although the reference PR/guide we create for the next 90ish products should likely do everything at once.

Reply:
Bit of scope inflation, but I understand the usefulness of doing so. We can add that to the migration. Would a simple `examples` folder within the product folder be sufficient?

Reply:
sgtm!